### DISCUSSION TITLE
Release 0.14.0

### DISCUSSION THREAD

**Discussion by rushtehrani** [onepanelio/core#679] [Announcements]
# 🚀  Onepanel 0.14.0 released!

Onepanel 0.14.0 has been released! It contains both new features and bug fixes. Some feature highlights include:

- New VS Code template with GPU, TensorFlow and PyTorch support
- AWS EKS deployment is now more automated, with auto scaling supported without additional configuration.
- Workflow node information panel now displays all associated artifacts.
- Labels are now visible across all resources in the platform
- Better authentication and RBAC support
- Workflow "run again" with ability to edit parameters
- More reliable Workspace resource management across all providers

See [Changelog](https://github.com/onepanelio/core/releases/tag/v0.14.0) for a complete list of new features and bug fixes.
